src/controllers/userController.ts

```diff
@@ -0,0 +1,34 @@
+import { Request, Response } from "express";
+import axios from "axios";
+import asyncRetry from "async-retry";
+import { User } from "../types/User";
+
+const transformUserData = (user: any): User => {
```

Comment:
Create a separate util class for this transform data and call it here in your controller to keep your controller as simple and small as possible.